scheds/rust/scx_layered/src/bpf/cost.bpf.c

```diff
@@ -26,7 +43,7 @@ struct {
 	__uint(type, BPF_MAP_TYPE_ARRAY);
 	__type(key, u32);
 	__type(value, struct cost);
-	__uint(max_entries, MAX_NUMA_NODES + 1);
+	__uint(max_entries, MAX_GLOBAL_BUDGETS);
```

Comment:
I think this can actually be 1 since there is only one global budget being used.